### DISCUSSION TITLE
Can we have row specific variables, And also want to able to copy a row from one dashboard to another dashboard

### DISCUSSION THREAD

**Discussion by vijay62723** [grafana/grafana#39997] [Dashboards, Panels & Visualizations]
As of now we can define variables for an entire dashboard, can we have row specific variables which only effects that particular row in the dashboard?
Also there is no option to copy the row to another dashboard, there is a copy option for panel and duplicate option to copy entire dashboard but it would be great if we have same type of option for entire row

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
